### DISCUSSION TITLE
Modification sur landing page ma cantine

### DISCUSSION THREAD

**Discussion by valeriemerle** [betagouv/ma-cantine#1870] [Idées]
- Le lien qui pointe vers la plaquette EGAlim est à mettre à jour avec la nouvelle version de la plaquette mise dans le gitbook
![Capture d’écran 2022-09-09 142600](https://user-images.githubusercontent.com/73835224/189349922-75bad5e1-8b88-413f-b70d-7a7e700dfa9d.png)
![Capture d’écran 2022-09-09 142649](https://user-images.githubusercontent.com/73835224/189349925-93e88133-0cd8-4748-8512-ce550483797d.png)
- Modifier le texte concernant la mesure "substitution du plastique"
[modif_site_mesuresEGAlim_plastique.docx](https://github.com/betagouv/ma-cantine/files/9535368/modif_site_mesuresEGAlim_plastique.docx)

**Comment by alemangui** [comment]
Hello Val :wave: 

- Peux-tu nous donner le lien du nouveau document stp ? J'ai regardé sur Gitbook mais je ne veux pas me tromper de doc
- Quel texte exactement veux-tu qu'on remplace ?

**Comment by valeriemerle** [comment]
Jenn l'a changé discussion close
